packages/graphiql/src/components/GraphiQL.tsx

```diff
@@ -116,12 +123,15 @@ export type GraphiQLState = {
   schema?: GraphQLSchema;
   query?: string;
   variables?: string;
+  headers?: string;
   operationName?: string;
   docExplorerOpen: boolean;
   response?: string;
   editorFlex: number;
-  variableEditorOpen: boolean;
-  variableEditorHeight: number;
+  extraEditorOpen: boolean;
```

Comment:
Saw it was in your todo, so not worried about it, but think it's important to have an easy way to disable the headers tab

Reply:
yes, `variableEditorOpen` prop should not be getting removed here, i thought it was only going to be deprecated?

Reply:
Do we need to leave the `variableEditorOpen` prop in the GraphiQLState type? I assumed we only needed to leave the `defaultVariableEditorOpen` prop for the GraphiQLProps type.

Reply:
ah, ok, in that case we're fine. i should have time to actually check this out locally tomorrow. been trying to finish up a big PR today

Reply:
Should the prop be `headerEditorEnabled` or `headerEditorDisabled`? And should it be enabled by default or not?